src/modules/mod.rs

```diff
@@ -19,6 +19,7 @@ pub mod script;
 pub mod sysinfo;
 #[cfg(feature = "tray")]
 pub mod tray;
+pub mod upower;
```

Comment:
This should have a cfg attr:

```suggestion
#[cfg(feature = "upower")]
pub mod upower;
```